scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1404,40 +1572,41 @@
                (sched/pending-jobs->considerable-jobs
                  (d/db conn) gpu-jobs user->quota user->usage num-considerable nil)))))))
 
-(deftest test-matches->job-uuids
-  (let [create-task-result (fn [job-uuid _ _ gpus]
-                             (-> (Mockito/when (.getRequest (Mockito/mock TaskAssignmentResult)))
-                                 (.thenReturn (sched/make-task-request
-                                                (Object.)
-                                                {:job/uuid job-uuid
-                                                 :job/resource (cond-> [{:resource/type :resource.type/mem, :resource/amount 1000.0}
-                                                                        {:resource/type :resource.type/cpus, :resource/amount 1.0}]
-                                                                       gpus (conj {:resource/type :resource.type/gpus, :resource/amount gpus}))}
-                                                nil
-                                                :task-id (str "task-id-" job-uuid)))
-                                 (.getMock)))
-        job-1 (create-task-result "job-1" 1 1024 nil)
-        job-2 (create-task-result "job-2" 2 2048 nil)
-        job-3 (create-task-result "job-3" 3 1024 1)
-        job-4 (create-task-result "job-4" 4 1024 nil)
-        job-5 (create-task-result "job-5" 5 2048 2)
-        job-6 (create-task-result "job-6" 6 1024 3)
-        job-7 (create-task-result "job-7" 7 1024 nil)]
-    (is (= #{"job-3" "job-5" "job-6"}
-           (sched/matches->job-uuids
-             [{:tasks [job-3]}, {:tasks #{job-5}}, {:tasks [job-6]}] nil)))
-    (is (= #{"job-1" "job-2" "job-4" "job-7"}
-           (sched/matches->job-uuids
-             [{:tasks [job-1 job-2]}, {:tasks #{job-4}}, {:tasks [job-7]}] nil)))
-    (is (= #{"job-1" "job-2" "job-4" "job-7"}
-           (sched/matches->job-uuids
-             [{:tasks [job-1 job-2]}, {:tasks #{job-4}}, {:tasks #{}}, {:tasks [job-7]}] nil)))
-    (is (= #{"job-3" "job-5" "job-6"}
-           (sched/matches->job-uuids
-             [{:tasks [job-3]}, {:tasks #{job-5}}, {:tasks #{job-6}}, {:tasks []}] nil)))
-    (is (= #{}
-           (sched/matches->job-uuids
-             [{:tasks []}, {:tasks #{}}, {:tasks #{}}, {:tasks []}] nil)))))
+(comment
```

Comment:
Was this deliberately disabled?

Reply:
no I think it was causing some issues before so I temporarily commented it out, but it seems to be working fine now

Reply:
uncommented